### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by schneidmaster** [schneidmaster/debatevid.io#11]
There are a few items raised by [this report](http://wave.webaim.org/report#/https://debatevid.io/) that should be addressed to improve accessibility for folks using screen readers and similar assistive technology.

**Comment by ryan-jr** [comment]
I'll take this one
